tests/foreman/api/test_syncplan.py

```diff
@@ -381,6 +397,32 @@ def test_positive_update_interval(self):
                     interval
                 )
 
+    @tier1
+    @run_only_on('sat')
+    def test_positive_update_interval_custom_cron(self):
+        """Create a sync plan and update its interval to custom cron.
+
+        :id: 26c58319-cae0-4b0c-b388-2a1fe3f22344
+
+        :expectedresults: A sync plan is created and its interval can be
+            updated to custom cron.
+
+        :CaseImportance: Critical
+        """
+        for interval in valid_sync_interval():
+            sync_plan = entities.SyncPlan(
+                description=gen_string('alpha'),
+                organization=self.org,
+                interval=interval
+            ).create()
+
+            sync_plan.interval = 'custom cron'
+            sync_plan.cron_expression = gen_choice(valid_cron_expressions())
+            self.assertEqual(
+                sync_plan.update(['interval', 'cron_expression']).interval,
+                'custom cron'
```

Comment:
perhaps we should add "custom cron" to SYNC_INTERVAL constant and use that.
constants.py
```python
...
SYNC_INTERVAL = {
    'hour': "hourly",
    'day': "daily",
    'week': "weekly",
    'custom': "custom cron",
}
...
```
and use it as SYNC_INTERVAL['custom']

Reply:
make sense! will update accordingly